<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispatchPane</class>
 <widget class="QWidget" name="dispatch_pane_">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="dispatch_pane_vertical_layout_">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="ScaledPushButton" name="dispatch_title_">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dispatch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ScaledLabel" name="user_marker_stack_">
     <property name="text">
      <string>user/marker/stack</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ScaledLabel" name="user_marker_">
     <property name="text">
      <string>User marker</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="loading_bar_spacer_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="DispatchLoadingBar" name="loading_bar_">
    </widget>
   </item>
   <item>
    <spacer name="loading_bar_bottom_spacer_">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="ScaledLabel" name="error_label_">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dispatch_region_">
     <layout class="QHBoxLayout" name="dispatch_layout_" stretch="5,5">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="traversal_counts_layout_" columnstretch="3,2,1">
        <item row="0" column="0">
         <widget class="ScaledLabel" name="traversal_counts_label_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignTop</set>
          </property>
          <property name="text">
           <string>Traversal statistics</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="ScaledLabel" name="rays_per_pixel_label_">
          <property name="text">
           <string>Rays per pixel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ScaledLabel" name="rays_per_pixel_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="ScaledLabel" name="ray_count_label_">
          <property name="text">
           <string>Total ray count</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="ScaledLabel" name="ray_count_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="ScaledLabel" name="loop_iteration_label_">
          <property name="text">
           <string>Traversal loop count</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="ScaledLabel" name="loop_iteration_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="ScaledLabel" name="instance_intersection_label_">
          <property name="text">
           <string>Instance intersections</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="ScaledLabel" name="instance_intersection_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="traversal_counts_spacer_">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="ScaledLabel" name="tlases_label_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignTop</set>
          </property>
          <property name="text">
           <string>TLASes traversed</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="tlases_horizontal_layout_">
         </layout>
        </item>
        <item row="8" column="0">
         <spacer name="tlases_spacer_">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0">
         <widget class="ScaledLabel" name="shader_invocations_label_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignTop</set>
          </property>
          <property name="text">
           <string>Shader invocations</string>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="ScaledLabel" name="raygen_label_">
          <property name="text">
           <string>Raygen</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="ScaledLabel" name="raygen_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="DispatchLegend" name="raygen_legend_">
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="ScaledLabel" name="closest_hit_label_">
          <property name="text">
           <string>Closest hit</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="ScaledLabel" name="closest_hit_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="DispatchLegend" name="closest_hit_legend_">
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="ScaledLabel" name="any_hit_label_">
          <property name="text">
           <string>Any hit</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="ScaledLabel" name="any_hit_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="DispatchLegend" name="any_hit_legend_">
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="ScaledLabel" name="intersection_label_">
          <property name="text">
           <string>Intersection</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="ScaledLabel" name="intersection_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="DispatchLegend" name="intersection_legend_">
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="ScaledLabel" name="miss_label_">
          <property name="text">
           <string>Miss</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="ScaledLabel" name="miss_content_">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="DispatchLegend" name="miss_legend_">
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="donut_layout_">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="donut_spacer_">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="DonutPieWidget" name="donut_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScaledLabel</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_label.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledPushButton</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_push_button.h</header>
  </customwidget>
  <customwidget>
   <class>DispatchLegend</class>
   <extends>QWidget</extends>
   <header>views/custom_widgets/dispatch_legend.h</header>
  </customwidget>
  <customwidget>
   <class>DonutPieWidget</class>
   <extends>QWidget</extends>
   <header>qt_common/custom_widgets/donut_pie_widget.h</header>
  </customwidget>
  <customwidget>
   <class>DispatchLoadingBar</class>
   <extends>QWidget</extends>
   <header>views/custom_widgets/dispatch_loading_bar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>

